src/test/kotlin/no/nav/familie/ef/sak/ekstern/EksternBehandlingControllerTest.kt

```diff
@@ -64,14 +76,54 @@ internal class EksternBehandlingControllerTest {
             behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(Stønadstype.OVERGANGSSTØNAD, setOf(ident1, ident2))
         } returns behandling(fagsak())
         assertThat(eksternBehandlingController.finnesBehandlingForPerson(Stønadstype.OVERGANGSSTØNAD, PersonIdent(ident1)).data)
-                .isEqualTo(true)
+            .isEqualTo(true)
     }
 
     @Test
     internal fun `uten stønadstype - skal returnere false når det ikke finnes noen behandling`() {
         every { behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(any(), setOf(ident1, ident2)) } returns null
         assertThat(eksternBehandlingController.finnesBehandlingForPerson(null, PersonIdent(ident1)).data)
-                .isEqualTo(false)
+            .isEqualTo(false)
+    }
+
+    @Test
+    internal fun `opprett ikke-utdaterte andeler som er maksimalt under ett år, forvent behandlinger utdaterte lik false`() {
+        val uuid1 = UUID.randomUUID()
+        val uuid2 = UUID.randomUUID()
+        val tilkjenteYtelser = opprettTilkjenteYtelser(false)
+        every {
+            behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(Stønadstype.OVERGANGSSTØNAD, any())
+        } returns behandling(id = uuid1)
+        every {
+            behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(Stønadstype.BARNETILSYN, any())
+        } returns behandling(id = uuid2)
+        every {
+            behandlingRepository.finnSisteBehandlingSomIkkeErBlankett(Stønadstype.SKOLEPENGER, any())
+        } returns null
+        every { tilkjentYtelseService.hentForBehandling(uuid1) } returns tilkjenteYtelser.elementAt(0)
+        every { tilkjentYtelseService.hentForBehandling(uuid2) } returns tilkjenteYtelser.elementAt(1)
```

Comment:
Synes det var litt vanskelig å skjønne att den ene skal returnere den første og den andre skal returnere den andre, burde man kanskje ha en støtte-metod som kun oppretter en tilkjent ytelse sånn att det er litt tydligere hva som returneres for de ulike behandlingene?